src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -139,8 +138,8 @@
         this.clientAuthType = ClientAuthType.CLIENT_AUTH_NONE;
 
         this.identificationProtocol = null;
-        this.serverNames = Collections.<SNIServerName>emptyList();
-        this.sniMatchers = Collections.<SNIMatcher>emptyList();
+        this.serverNames = Collections.emptyList();
+        this.sniMatchers = Collections.emptyList();
```

Comment:
Here you use Collections.emptyList() instead of List.of(). Probably ok, but maybe you want to change that to List.of().

Reply:
There are some other places that use Collections.emptyList(), I will make the update all together in another bug.